samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -25,6 +25,7 @@ LOG_MODULE_REGISTER(fp_sample, LOG_LEVEL_INF);
 
 #define FP_ADV_MODE_BUTTON_MASK					DK_BTN1_MSK
 #define VOLUME_UP_BUTTON_MASK					DK_BTN2_MSK
+#define BOND_REMOVE_BUTTON_MASK					DK_BTN3_MSK
```

Comment:
shouldn't you described the UI interface addition in the sample Readme file?

Reply:
Doc update will be done in a separate PR (to limit PR's scope and wait for approvals before updating doc)